.github/workflows/main-publish.yml

```diff
@@ -0,0 +1,32 @@
+# PyPi publish for SubmitCGAP 
+
+name: publish 
```

Comment:
How is this invoked? Is there a menu somewhere of actions? Is this on the menu? Should these have uppercase initial? Or is this a name used in code like a function/variable would be?